src/main/java/se/sundsvall/document/api/model/DocumentDataCreateRequest.java

```diff
@@ -17,6 +18,9 @@ public class DocumentDataCreateRequest {
 	@Schema(description = "Confidentiality")
 	private Confidentiality confidentiality;
 
+	@Schema(description = "Should the document be archived?", example = "false")
+	private Boolean archive;
```

Comment:
Change to boolean as this should be either true or false on create. Only update should be able to set null, right?